qa/L0_http/test.sh

```diff
@@ -467,9 +467,10 @@ kill $SERVER_PID
 wait $SERVER_PID
 
 # Run cpp client unit test
-rm -r ${MODELDIR}/* && cp -r $DATADIR/qa_model_repository/onnx_int32_int32_int32 ${MODELDIR}/.
```

Comment:
Should keep the `rm` and re-construct the model repository as sanity check that the model repository will be in expected structure.

Reply:
Updated, please review, thank you